index.html

```diff
@@ -3484,6 +3484,29 @@ <h3>Media Types</h3>
 those other syntaxes to be transformed to the `application/vc+ld+json`
 media type.
         </p>
+
+        <p>
+Additional `application/vc+` media type transformation specifications can be
+defined to provide implementers with further options for producing objects
+compliant with the base media type.
+
```

Comment:
I think this will be much clearer to the reader if the paragraph breaks in this HTML source are preserved for final rendering, so this should get a few more instances of `</p><p>`.